packages/frames.js/src/core/utils.ts

```diff
@@ -47,6 +47,14 @@ function isValidButtonAction(action: unknown): action is ButtonActions {
   return typeof action === "string" && action in buttonActionToCode;
 }
 
+function isUrlObjectComplete(urlObject: UrlObject): boolean {
+  return (
```

Comment:
You should also check if the values are set right? URLObject is just saying that there are properties but their values can be empty right?

Also wouldn't it make sense to also allow URL class instance?

Reply:
Tha last point about URL instance is for generateTargetUrl function. baseUrl is typed like that.

Reply:
`URLObject` is already compatible with `URL`, the user is able to pass an `URL` object. Also to add `URL` explicitly the whole call stacks will be updated. as it is gaining nothing I think let's leave that part out? 

I've updated value check into `isUrlObjectComplete`, pls have a look @michalkvasnicak , thanks!